tests/test_cloudwatch/test_cloudwatch.py

```diff
@@ -51,6 +53,7 @@ def test_create_alarm():
     list(alarm.ok_actions).should.equal(["arn:ok"])
     list(alarm.insufficient_data_actions).should.equal(["arn:insufficient"])
     alarm.unit.should.equal("Seconds")
+    alarm.alarm_arn.should.equal(make_arn_for_alarm("us-east-1", ACCOUNT_ID, "tester"))
```

Comment:
You might not want to couple this check so tightly to the underlying implementation.  If a future version of `moto` supports, say, a user-definable account id, this test will break.  You really just want to make sure the alarm arn is in the response.  You could go a bit further and check that it starts with `arn` and contains the `tester` resource id.  That would be a perfectly adequate test without importing an implementation detail (`make_arn_for_alarm`).

Reply:
Agreed